### PR TITLE
Feature/add footer with links 69 70

### PR DESCRIPTION
**Pull request by abdullah-rana**
- Added footer on index, search results, and detail pages
- Impressum and Datenschutz pages are not hosted locally. These links will redirect the user to NFDI4DS pages for impressum and datenschutz.

### CODE REVIEW

**Comment by RicardoUsbeck** [comment]
I noticed that a lot of files just have formatting changes. Make sure not to autoformat too much

**Comment by janreineke** [comment]
If I remember right, @RicardoUsbeck mentioned in our last meeting that we should link imprint and privacy statement to the UHH?
https://www.uni-hamburg.de/en/impressum.html
https://www.uni-hamburg.de/en/datenschutz.html

File: templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                </div>
+                <div class="col-lg-4 col-md-6">
+                    <div class="f_widget about-widget pl_70 wow fadeInLeft" data-wow-delay="0.6s"
+                        style="visibility: visible; animation-delay: 0.6s; animation-name: fadeInLeft;">
+                        <h3 class="f-title f_600 t_color f_size_18">Connect</h3>
+                        <ul class="list-unstyled f_list">
+                            <li><a href="https://www.linkedin.com/company/nfdi4ds/">Linkedin</a></li>
+                            <li><a href="https://github.com/NFDI4DS">Github</a></li>
+                            <li><a href="https://twitter.com/nfdi4ds">Twitter</a></li>
+                        </ul>
```
Comment (huntila) [inline templates/footer.html:37]: Make target='_blank' to all the links.

**Comment by abdullah-rana** [comment]
> I noticed that a lot of files just have formatting changes. Make sure not to autoformat too much

@RicardoUsbeck Noted, I have disabled my local formatter to avoid autoformatting. 

> If I remember right, @RicardoUsbeck mentioned in our last meeting that we should link imprint and privacy statement to the UHH? https://www.uni-hamburg.de/en/impressum.html https://www.uni-hamburg.de/en/datenschutz.html

@janreineke Thanks Jan, yes it was discussed, but I thought UHH pages are just for the reference to see what to add inthese new pages. We being part of NFDI p

**Pull request merged**